main_routers/characters_router.py

```
@@ -330,33 +359,178 @@ async def update_catgirl_l2d(name: str, request: Request):
+        characters['猫娘'][name]['lighting'] = {
+            'ambient': float(lighting['ambient']),
+            'main': float(lighting['main']),
+            'fill': float(lighting['fill']),
+            'rim': float(lighting['rim'])
+        }
+        logger.info(f"已保存角色 {name} 的打光配置: {characters['猫娘'][name]['lighting']}")
+
+        # 保存配置到文件
+        _config_manager.save_characters(characters)
+        
+        # 轻量级更新：直接更新运行时内存状态（如果存在）
```

Ruff (0.14.10)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)